src/main/java/seedu/loyaltylift/ui/Badge.java

```diff
@@ -63,7 +63,17 @@ public static Badge createCustomerTypeBadge(CustomerType customerType) {
      * @return An instance of the Badge.
      */
     public static Badge createPointsBadge(Points points) {
-        return new Badge(Color.valueOf("#f09030"), Color.WHITE, points.value.toString() + " pts");
+        Points.Tier tier = points.getLoyaltyTier();
+        switch (tier) {
+        case GOLD:
+            return new Badge(Color.valueOf("#d4af37"), Color.WHITE, points.value.toString() + " pts");
+        case SILVER:
+            return new Badge(Color.valueOf("#808080"), Color.WHITE, points.value.toString() + " pts");
+        case BRONZE:
+            return new Badge(Color.valueOf("#CD7F32"), Color.WHITE, points.value.toString() + " pts");
+        default:
```

Comment:
Instead of using default as a fallback to NONE tier. I feel that it is safer to use an additional 'case' statement here for NONE tier. The default should throw an exception instead